spec/javascripts/Users/permissions.spec.js

```diff
@@ -13,7 +13,7 @@ import {
   AdminSection,
   ServiceAccess,
   Form
-} from 'Users/permissions'
+} from '../../../app/javascript/src/Users/permissions'
```

Comment:
@didierofrivia Isn't there a simpler and less breakable way to reference to `Users/permissions`?

Reply:
This is wrong, please use the absolute path.

Reply:
Oookay, I got it @Martouta @josemigallas 
When I rebased on top of master, it was already there: https://github.com/3scale/porta/commit/71d256cd4a68611d5e4a5ed8c3858de1841a5cb4#diff-9a5814ce9a4f85d0eb380f2e8aa6dc65R16

I didn't want to change it, because it was not my piece. But seems that I missed https://github.com/3scale/porta/commit/08ff8aff0b17fbf2827d0143a0b3a22f7cbd3623#diff-9a5814ce9a4f85d0eb380f2e8aa6dc65R16

Will change.